src/p_spec.c

```diff
@@ -2644,24 +2644,18 @@ void T_Scroll(scroll_t *s)
         side = sides + s->affectee;
         side->basetextureoffset += dx;
         side->baserowoffset += dy;
-        side->textureoffset = side->basetextureoffset;
```

Comment:
I don't think this part of the code is problematic. That is, I think it is alright to reset the offset values to the corresponding base offsets once every tic in `T_Scroll()`.